.github/workflows/publish.yml

```diff
@@ -16,7 +16,8 @@ jobs:
       - name: Install Hugo
         uses: peaceiris/actions-hugo@v2
         with:
-          hugo-version: "0.113.0"
+          hugo-version: "0.119.0"
```

Comment:
Image processing was released in v0.119.0. Hugo requires the [extended version](https://gohugo.io/troubleshooting/faq/#i-get-this-feature-is-not-available-in-your-current-hugo-version) be used for this feature.

Reply:
We were due for an upgrade anyway; any reason not to go all the way to v0.138.0?

Reply:
I've upgraded to the latest Hugo version and there's an issue with the child menu links not working as expected. If I can figure that out quickly then we should be able to upgrade to the latest version without an issue.